app/views/foreman_ansible/job_templates/ansible_windows_updates.erb

```diff
@@ -0,0 +1,187 @@
+<%#
+name: Ansible Windows - Updates
+snippet: false
+template_inputs:
+- name: blacklist
+  required: false
+  input_type: user
+  description: "A list of update titles or KB numbers that can be used to specify
+    which updates are to be excluded from installation.\r\nIf an available update
+    does match one of the entries, then it is skipped and not installed.\r\nEach entry
+    can either be the KB article or Update title as a regex according to the PowerShell
+    regex rules."
+  advanced: false
+  value_type: plain
+  resource_type: Katello::ActivationKey
+  hidden_value: false
+- name: category_names
+  required: false
+  input_type: user
+  description: "A scalar or list of categories to install updates from. To get the
+    list of categories, run the module with state=searched. The category must be the
+    full category string, but is case insensitive.\r\nSome possible categories are
+    Application, Connectors, Critical Updates, Definition Updates, Developer Kits,
+    Feature Packs, Guidance, Security Updates, Service Packs, Tools, Update Rollups
+    and Updates.\r\n\r\nDefault is '[\"CriticalUpdates\", \"SecurityUpdates\", \"UpdateRollups\"]'"
+  advanced: false
+  value_type: plain
+  resource_type: Katello::ActivationKey
+  hidden_value: false
+- name: log_path
+  required: false
+  input_type: user
+  description: If set, win_updates will append update progress to the specified file.
+    The directory must already exist.
+  advanced: false
+  value_type: plain
+  resource_type: Katello::ActivationKey
+  hidden_value: false
+- name: reboot
+  required: false
+  input_type: user
+  description: "Ansible will automatically reboot the remote host if it is required
+    and continue to install updates after the reboot.\r\nThis can be used instead
+    of using a win_reboot task after this one and ensures all updates for that category
+    is installed in one go.\r\nAsync does not work when reboot=true.\r\n\r\nDefault
+    is 'false'"
+  options: "false\r\ntrue"
+  advanced: false
+  value_type: plain
+  resource_type: Katello::ActivationKey
+  hidden_value: false
+- name: reboot_timeout
+  required: false
+  input_type: user
+  description: "The time in seconds to wait until the host is back online from a reboot.\r\nThis
+    is only used if reboot=true and a reboot is required.\r\n\r\nDefault is '1200'"
+  advanced: false
+  value_type: plain
+  resource_type: Katello::ActivationKey
```

Comment:
I don't think that the resource_type should be set like this.

Reply:
Fixed in second commit